### DISCUSSION TITLE
小部件的长宽尺寸能改变吗？

### DISCUSSION THREAD

**Discussion by Agnon1573** [bluecaret/carettab#370] [Ideas and Feature Requests]
小部件的尺寸大小的固定的，无法进行动态调整；比如引言小部分，想要横向尺寸稍微大点，但无法调整

**Comment by bluecaret** [comment]
Hello. You can adjust the size of widgets by going to the widget settings, and opening the "Widget container" section, then editing the options under "Container alignment and size"

Translated with Google Translate:
你好。 您可以通过转到小部件设置并打开“小部件容器”部分，然后编辑“容器对齐和大小”下的选项来调整小部件的大小

![image](https://github.com/bluecaret/carettab/assets/9217636/31cbe47f-e57c-482f-8220-4cd49e6b8ee7)
